src/bun.js/bindings/bindings.cpp

```diff
@@ -4326,14 +4326,15 @@ static void fromErrorInstance(ZigException* except, JSC::JSGlobalObject* global,
                 iterator.forEachFrame([&](const V8StackTraceIterator::StackFrame& frame, bool& stop) -> void {
                     ASSERT(except->stack.frames_len < frame_count);
                     auto& current = except->stack.frames_ptr[except->stack.frames_len];
+                    current = {};
 
                     String functionName = frame.functionName.toString();
                     String sourceURL = frame.sourceURL.toString();
                     current.function_name = Bun::toStringRef(functionName);
                     current.source_url = Bun::toStringRef(sourceURL);
-                    current.position.line = frame.lineNumber.zeroBasedInt();
-                    current.position.column_start = frame.columnNumber.zeroBasedInt();
-                    current.position.column_stop = frame.columnNumber.zeroBasedInt();
+                    current.position.line = frame.lineNumber.oneBasedInt();
```

Comment:
We can't use `oneBasedInt()` because we expected them to be zero based when they get to `SourceURLFormatter.format()`: https://github.com/oven-sh/bun/blob/94bf404c417f2b6ecb716679d86ed47e26fb157a/src/bun.js/bindings/exports.zig#L621-L640

Reply:
I think this looks like somewhere we are mixing up one/zero based ints when going to and from JSC, in Bun::formatStackTrace we are converting it into a oneBasedInt: https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/bun.js/bindings/ZigGlobalObject.cpp#L383-L393